static/app/views/insights/sessions/queries/useErroredSessions.tsx

```diff
@@ -42,25 +39,18 @@ export default function useErrorFreeSessions() {
     {staleTime: 0}
   );
 
-  if (isPending || !sessionData) {
-    return {
-      series: [],
```

Comment:
Does this affect the charts at all, looks like now we'll always have at least a series w/ length 1 and potentially empty data

Reply:
ph, derp. I was experimenting with a hook lower down from here and didn't relalize. I'll put it back.